packages/app-mobile/components/NoteList.tsx

```diff
@@ -47,42 +64,22 @@ class NoteListComponent extends Component {
 		return this.styles_[themeId];
 	}
 
-	createNotebookButton_click() {
+	private createNotebookButton_click() {
 		this.props.dispatch({
 			type: 'NAV_GO',
 			routeName: 'Folder',
 			folderId: null,
 		});
 	}
 
-	filterNotes(notes) {
```

Comment:
Removed this function since it doesn't seem to be used anywhere in the code.

Reply:
Makes sense, thanks for this